tgui/packages/tgui-dev-server/package.json

```diff
@@ -1,13 +1,16 @@
 {
   "private": true,
   "name": "tgui-dev-server",
-  "version": "5.0.0",
+  "version": "5.0.3",
   "type": "module",
   "dependencies": {
     "axios": "^1.8.2",
-    "glob": "^7.1.7",
+    "glob": "^7.2.0",
```

Comment:
really no point in upgrading this but sure https://github.com/isaacs/node-glob/compare/v7.1.7...v7.2.0